README.adoc

```diff
@@ -166,11 +173,12 @@ The [hotspot=maven-failsafe-plugin]`maven-failsafe-plugin` artifact runs your Ar
 
 Lastly, specify the [hotspot=liberty-maven-plugin]`liberty-maven-plugin`
 configuration that defines your Open Liberty runtime configuration.
-The `configure-arquillian` goal configures your Arquillian container.
-This goal automatically retrieves core server parameters from the `liberty-maven-plugin` configuration
-and enables you to customize your Arquillian container configuration directly in your POM file if necessary.
-You can learn more about the `configure-arquillian` goal in its
-https://github.com/WASdev/ci.maven/blob/master/docs/configure-arquillian.md[documentation^].
+When the application runs in the Arquillian Liberty managed container, 
+the context root of the application will be assigned as same as the war file name.
```

Comment:
The wording here should be corrected:
>...the context root of the application will be assigned as same as the war file name

Could be corrected to:
>...the name of the war file will be used as the context root of the application.

Reply:
@griffinhadfield I updated